src/ui/map/util.js

```diff
@@ -113,9 +126,16 @@ const addMarkers = (geojson, context, writable) => {
       }
     }
 
+    let symbol = 'circle'
+    if (d.properties && d.properties['marker-symbol']) {
+        symbol = d.properties['marker-symbol']
+    }
+
     const marker = new ClickableMarker({
       color,
-      scale
+      scale,
+      symbol,
+      symbolColor
```

Comment:
`symbol-color` is not included in the `simplestyle-spec` https://github.com/mapbox/simplestyle-spec/tree/master/1.1.0

I'm fine adding it, but we should document this.  Can you add a section to `help.md` called "I want to style features" mentioning that geojson.io supports an extended version of simplestyle-spec, and explain what all the options are, and link to the simplestyle-spec repo?